requirements.txt

```diff
@@ -3,3 +3,4 @@ black==24.2.0
 flake8-annotations==3.0.1
 pylint==3.0.3
 pymap3d
+pyserial
```

Comment:
we should rearrange the requirements.txt to have the linters and formatters last - also it's a good idea to separate the dependencies with comments that are categories, so it could be 

\# Packaged in alphabetic order
pymap3d
pyserial

\# Linters and formatters are explicitly versioned
black==24.2.0
flake8-annotations==3.0.1
pylint==3.0.3